tools/MVP/templates/test_presenter.py.in

```diff
@@ -0,0 +1,34 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+from unittest import mock
+
+from model import Model
+from view import View
+from presenter import Presenter
+
+
+class TestPresenter(unittest.TestCase):
+
+    def setUp(self) -> None:
+        mock_view = mock.create_autospec(View)
+        mock_model = mock.create_autospec(Model)
+
+        self.presenter = Presenter(mock_model, mock_view)
+        mock_view.subscribe_presenter.assert_called_once()
+
+    def test_handle_button_clicked_will_update_model_count_and_set_the_view_label(self):
```

Comment:
Can the contents of this test be split (by spaces) into `Given-When-Then`? https://pythontest.com/strategy/given-when-then-2/

I find this way of structuring tests make them easier to read